### DISCUSSION TITLE
Audiogridder automation not working in parallels 18

### DISCUSSION THREAD

**Discussion by sloofin** [apohl79/audiogridder#1135] [Help] [answered]
After upgrading to parallels 18, any audiogridder plugin receiving automation will cause overloads in any DAW i try it in.
If windows is visible and full screen, there are no problems. As soon as windows isn't front and centre, any automation to any audiogridder plugin will cause whatever DAW i'm using to glitch and crackle. Even with an empty project in Logic and one synth, as soon as automation was added the audio engine would crackle and lose sync. This happens while the automation (or modulation in Bitwig's case) is moving, and doesn't when automation is still. This wasn't an issue at all on parallels 17. This happens with older versions of windows on parallels 18 too.

This is with audiogridder 1.2 beta 15.

Would also like to add to the chorus of people unable to effectively load vst2 presets (they show up in the list but don't change any parameters in their target plugins), and vst3 preset menu remains blank.

**Comment by apohl79** [comment, marked as answer]
Sounds like parallels is pausing background processes when not being in foreground on macos. Maybe you can disable that behavior in parallels.

**Reply by sloofin** [reply]
Thanks for your answer Andreas :) 

So after further digging it turned out to be the choice of hypervisor in parallels. With Parallel's adaptive hypervisor audiogridder would glitch if windows wasn't to the front in fullscreen. Switching off adaptive hypervisor resolved this problem and I'm now able to run audiogridder with automation and low buffers across various DAWs.

Logic has an additional problem though - Bitwig and Live were behaving, but Logic still glitched as soon as I automated parameters in audiogridder. It wasn't as bad as the hypervisor glitching but it still made it unusable. In Logic's Audio settings page on the General tab there's an option called Sample Accurate Automation. With this set to "Volume, Pan, Sends, Plugin parameters" audiogridder will glitch whenever receiving automation. This problem disappears when choosing "Volume, Pan, Sends" instead. I now have a happy and efficient audiogridder setup again :)

So the only problem remaining is the vst presets - VST2 presets appear when right clicking on the plugin in Audiogridder, but selections don't transmit parameter changes to the plugin. In VST3 presets are mostly greyed out, but when they are offered they exhibit the same behaviour - changing preset doesn't change anything on the plugin itself.

Thanks again for your reply - hope the above information is useful.

**Reply by apohl79** [reply]
thats great info, thx! will try to reproduce the automation issue in logic. regarding the presets, I filed a bug and will investigate that.
